app/assets/stylesheets/common.scss.erb

```diff
@@ -90,14 +90,17 @@
     .#{$namespace}-download-count {
       display: inline-block;
       min-width: 18px;
-      padding-top: 2px;
-      padding-bottom: 2px;
-      line-height: 1;
       color: $color-pantone-405;
-      background-color: #fff;
+      background-color: $btn-default-bg;
       text-align: center;
-      vertical-align: middle;
+      vertical-align: text-top;
       border-radius: 16px;
+      font-size: 12px;
+      line-height: 16px;
+      margin-left: 2px;
+      margin-bottom: 1px;
+      padding-left: 3px;
+      padding-right: 3px;
```

Comment:
Since we're updating so much of the declarations in this selector, can we re-order these to be alpha? (which is the preferred scss convention).